### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'int java.lang.Enum.ordinal()' on a null object reference

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#14851]
**Sentry Url**: https://sentry.io/share/issue/6ec5db78e4714d53b50c7a650009a5b4/
**User Count**: 38
**Count**: 95
**First Release**: org.wordpress.android@17.5-rc-1+1056
**First Seen**: 2021-06-08T16:35:55.004000Z
**Last Seen**: 2021-06-16T10:41:20Z
**24 Hours**: 51
**30 Days**: 95

```
NullPointerException: Attempt to invoke virtual method 'int java.lang.Enum.ordinal()' on a null object reference
     at org.wordpress.android.util.WPMediaUtils.getErrorMessage(WPMediaUtils.java:185)
     at org.wordpress.android.ui.uploads.UploadUtils.getErrorMessageFromMediaError(UploadUtils.java:151)
     at org.wordpress.android.ui.uploads.UploadUtils.getErrorMessageFromMedia(UploadUtils.java:180)
     at org.wordpress.android.ui.posts.EditPostActivity.getErrorMessageFromMedia(EditPostActivity.java:3625)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment.showRetryMediaUploadDialog(GutenbergEditorFragment.java:752)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment.access$1400(GutenbergEditorFragment.java:82)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment$1.onRetryUploadForMediaClicked(GutenbergEditorFragment.java:271)
...
(9 additional frame(s) were not displayed)
```

**Comment by ashiagr** [comment]
Looking at the stack trace, seems related to #14852 but with a higher occurrence.

I tried the retry media upload flow but could not reproduce the issue. Since the issue is recently introduced and numbers are increasing, affecting the editing flow, assigning it a medium priority.

**Comment by ashiagr** [comment]
Closed via #14858.
